apps/gitness/src/pages/repo/repo-branch-list.tsx

```diff
@@ -121,9 +122,9 @@ export function ReposBranchesListPage() {
             },
             //hardcoded
             checks: {
-              done: 1,
-              total: 1,
-              status: 1
+              done: undefined,
+              total: undefined,
+              status: undefined
             },
```

Comment:
Let's make `checks` optional, as it's possible to not have any checks on a branch.

Reply:
fixed